File: speed_control.py
def gps(s, x):
    arr = []
    if len(x) <= 1:
        return 0
    for item in range(len(x)-1):
        arr.append( (x[item+1] - x[item]) * 3600 / s)
    biggest = max(arr)
    return biggest

File: speedControl.js
function gps(s, x) {
    let arr = [];
  if (x.length <=1){
    return 0;
  }
  for(let i = 0; i < x.length -1; i++){
   arr.push((x[i+1] - x[i]) * 3600 / s)
  }
 let biggest = Math.max(...arr)
 return biggest;
}
